Disable too strict check on version of AUGUSTUS and the GeneMark key in the
home directory of the user
author: Alex Domingo (Vrije Universiteit Brusel)
--- set_dependencies.py.orig	2023-03-16 15:50:03.802522000 +0100
+++ set_dependencies.py	2023-03-16 15:50:28.562603082 +0100
@@ -284,15 +284,15 @@
     check_working_internal(samtools_path, [samtools_path, '--help'])
     check_working_internal(bamtools_path, [bamtools_path, '--help'])
     check_working_internal(augustus_path, [augustus_path, '--help'])
-    check_augustus_version(augustus_path)
+    # check_augustus_version(augustus_path)
 
-    # For GeneMark, check the .gm_key
-    home_path = os.path.expanduser('~')
-    if not os.path.exists(os.path.join(home_path, '.gm_key')):
-        sys.exit(
-            '\n[ERROR] You do not have .gm_key in your home directory.\n'
-            'Check https://wiki.gacrc.uga.edu/wiki/GeneMark'
-        )
+    # # For GeneMark, check the .gm_key
+    # home_path = os.path.expanduser('~')
+    # if not os.path.exists(os.path.join(home_path, '.gm_key')):
+    #     sys.exit(
+    #         '\n[ERROR] You do not have .gm_key in your home directory.\n'
+    #         'Check https://wiki.gacrc.uga.edu/wiki/GeneMark'
+    #     )
 
 def check_augustus_version(augustus_path):
     '''Check Augustus version 3.3.3'''
